src/gurobi_optimods/opf/converters.py

```diff
@@ -195,7 +214,7 @@ def convert_case_to_internal_format(case_dict):
             branch["ratio"] = 1.0
 
     alldata = {"LP": {}, "MIP": {}}
-
+    alldata["casename"] = case_dict["casename"]
```

Comment:
Add a test that "casename" comes through in the result (assuming that's the intention?)